elastic/kibana
# [Uptime] Decrypt Synthetics Monitors on sync and ensure API key is space agnostic

## Summary

Fixes https://github.com/elastic/kibana/issues/130313
Fixes https://github.com/elastic/kibana/issues/130312

Ensures that monitors are able to be decrypted from all spaces during the synthetics sync task.

Ensures that the Synthetics API Key can be accessed from all spaces by changing the saved object type from `single` to `agnostic`.

### Testing
The overall goal is to ensure that all monitors, created in any space, are able to be indexed appropriately. Likewise, enabling and disabling Monitor Management should work consistently from any space. 

1. Add the following keys to Kibana.yml https://p.elstc.co/paste/-uRAA0Lw#b4xgrKoWb6yEPQgW+JitoB4WN5eTLV8hLEEaavaV4qA
2. Navigate to Monitor Management in Uptime. 
3. Enable monitor management
4. Create a monitor
5. Navigate to Uptime. Ensure that the monitor is indexed appropriately
6. Create a second space 
7. Navigate to that second space
8. Create a monitor in the second space
9. Navigate to Uptime. Ensure that the monitor is indexed appropriately
10. Navigate to Monitor Management from the second space. Disable Monitor Management. Ensure that disabling monitor management is successful
11. Enable Monitor Management again from the second space. Ensure that enabling monitor management is successful.
12. Create another monitor in the second space.
13. Navigate to Uptime, ensure that the monitor is indexed successful
14. Navigate back to the default space. Ensure that the original monitor you created in the default space is still running
15. Create another monitor in the default space
16. Navigate to Uptime. Ensure that the monitor is indexed appropriately.
Summarize your PR. If it involves visual changes include a screenshot or gif.
